utils/wasm-proc/src/main.rs

```diff
@@ -114,6 +114,8 @@ const RT_ALLOWED_IMPORTS: [&str; 76] = [
     // From `GearBls12_381`
     "ext_gear_bls_12_381_aggregate_g1_version_1",
     "ext_gear_bls_12_381_map_to_g2affine_version_1",
+    // From `PoseidonHash`
+    "ext_poseidon_hash_poseidon_version_1",
```

Comment:
mb rename to `ext_gear_poseidon_hash_poseidon_version_1`? like `GearBls12_381` does

Reply:
Makes sense